onc_certification_g10_test_kit.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'rubyzip', '~> 2.3.2'
   spec.add_runtime_dependency 'smart_app_launch_test_kit', '0.1.5'
   spec.add_runtime_dependency 'tls_test_kit', '0.1.1'
-  spec.add_runtime_dependency 'us_core_test_kit', '0.2.5'
+  # spec.add_runtime_dependency 'us_core_test_kit', '0.2.5'
```

Comment:
Do we need this line or not?

Reply:
We'll be restoring it before we do a real release.